### PR TITLE
#275: add matomo tag manager

### PR DESCRIPTION
**Pull request by samrichca**

### CODE REVIEW

File: gatsby-browser.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
+import React from 'react';
 import scrollToElement from 'scroll-to-element';
+import { Layout } from 'components';
 
-export { wrapPageElement, wrapRootElement } from './gatsby-shared';
+export function wrapPageElement({ element, props }) {
```
Comment (samrichca) [inline gatsby-browser.js:5]: reverting changes from previous PR

File: src/templates/index.js [hunk 2]
```diff
@@ -3,18 +3,12 @@
...
 import PropTypes from 'prop-types';
-import Helmet from 'react-helmet';
 import { Footer, NavBar } from 'components';
-import config from 'meta/config';
 import 'styles/main.scss';
 
 export default function TemplateWrapper({ children }) {
   return (
     <div>
-      <Helmet>
```
Comment (samrichca) [inline src/templates/index.js:14]: remove redundant helmet tag

File: gatsby-node.js [hunk 3]
```diff
@@ -124,22 +124,21 @@ const createMarkdownPages = async ({ actions, graphql }) => {
...
-          component: path.resolve('src/templates/documentation/index.js'),
-          context: {
-            // use for graphQL query in template
-            id,
-            sectionSlug,
-          },
-        });
-      }
+
+      actions.createPage({
```
Comment (samrichca) [inline gatsby-node.js:133]: slug match for documentation wasn't needed

File: src/components/Layout/index.js [hunk 4]
```diff
@@ -100,12 +101,18 @@ class TemplateWrapper extends Component {
     const desktopMegaMenuCheck =
       typeof window !== 'undefined' && !mobileMenuOpen && window.innerWidth > 1160;
     const isDocs = path.includes('/documentation/') && data.mdx;
+    const matomoTagManager = getMatomoTagManager();
```
Comment (samrichca) [inline src/components/Layout/index.js:104]: I put the string interpolation for MTM in a separate file for organization. can't make a script tag component and import it into helmet.

**Review by daniel-cy-lu — APPROVED** [review]

**Pull request merged**